libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -1436,6 +1673,34 @@ AP_GPS_UBLOX::_configure_valget(ConfigKey key)
     return _send_message(CLASS_CFG, MSG_CFG_VALGET, &msg, sizeof(msg));
 }
 
+/*
+ *  configure F9 based key/value pair for a complete config list
+ */
+bool
+AP_GPS_UBLOX::_configure_config_set(const config_list *list, uint8_t count, uint32_t unconfig_bit)
+{
+    if (_hardware_generation < UBLOX_F9) {
```

Comment:
This is already protected before we get here, do we really need to run it twice?

Reply:
removed